README.md

```diff
@@ -1,34 +1,50 @@
-![CI](https://github.com/nearform/hub-template/actions/workflows/ci.yml/badge.svg?event=push)
+[![Continuous Integration](https://github.com/nearform/playwright-firebase/actions/workflows/ci.yml/badge.svg)](https://github.com/nearform/playwright-firebase/actions/workflows/ci.yml)
 
-# Playwright-Firebase plugin
+# @nearform/playwright-firebase
 
 Tidy way to authenticate Playwright E2E tests on Firebase.
 
-## Installation
+Install using npm:
 
-In order to install this package into your playwright project use `npm install @nearform/playwright-firebase` or `yarn add @nearform-firebase`
+```bash
+npm install @nearform/playwright-firebase
+```
+
+or yarn:
+
+```bash
+yarn add @nearform/playwright-firebase
+```
+
+Want to see it in action? Go to [Example](#example) to try it out!
+
+## Contents
 
-## Commands
+0. [Setup](#setup)
+1. [Motivation](#motivation)
+2. [Example](#example)
 
-- `auth.login(page)` : logs in
-- `auth.logout(page)`: logs out
+<a name="setup"></a>
 
 ## Setup
 
-### TypeScript
+### Firebase environment variables
 
-If you're using Typescript, one small addition you'll need to make is to add the type `Credentials` to your `playwright.config.ts` such that
+To set up this plugin you will need three environment variables in order to authenticate with Firebase. These are:
 
-```
-import {Credentials} from '@nearform/playwright-firebase'
-export default defineConfig<Credentials>({
-  ...
-  })
-```
+1. [Firebase Service Account](https://firebase.google.com/docs/app-distribution/authenticate-service-account)
+2. [Firebase User ID](https://firebase.google.com/docs/auth/web/manage-users)
+3. [Firebase Configurations](https://support.google.com/firebase/answer/7015592?hl=en#zippy=%2Cin-this-article)
 
-Create a setup file that is ran before all tests, where we'll redefine test, so you can import it from your setup file with the `auth` fixture added.
+For more information about Firebase you can read the documentation [here](https://firebase.google.com/docs/auth/web/start)
 
-```
+It's recommended to place these values in a `.env` file. Make sure to **NOT COMMIT THIS FILE**. For clarity, the Firebase User ID is often abbreviated to UID, as you will find below.
+
+### Attaching playright-firebase as a fixture to Playwright
+
+Playwright is based on fixtures. You have likely already used them within playwright, they are the `{ page }` object that is passed in to `test`. More information on them [here](https://playwright.dev/docs/test-fixtures). In the very same way, we are able to add our own fixture, which we call `auth` to the tests. To do so, we need to create a setup file that will automatically run before all other tests. We will call this `auth.setup.ts`
```

Comment:
Always spell Playwright correctly. Capital P in here (the only exception is when it occurs in the context of the package name)